Mathlib/Order/Antisymmetrization.lean

```diff
@@ -233,3 +247,22 @@ theorem OrderIso.dualAntisymmetrization_symm_apply (a : α) :
   rfl
 
 end Preorder
+
+section Prod
+
+variable (α β) [Preorder α] [Preorder β]
+
+/-- The antisymmetrization of a product preorder is isomorphic
+to the product of antisymmetrizations. -/
+def Antisymmetrization.prodEquiv :
```

Comment:
Does it make sense to provide `simp` lemmas here? e.g. `Antisymmetrization.prodEquiv ⟦x⟧ = (⟦x.1⟧, ⟦x.2⟧)`, etc.

Reply:
added two lemmas